scripts/lib/wic/plugins/source/acrn-bootimg-efi.py

```diff
@@ -0,0 +1,217 @@
+#
```

Comment:
My only concern here is that this code is more than just a slim customisation layer, can it not inherit from a class in oe-core or is some refactoring planned?

Reply:
refactoring needed once it comes to support features like pre-launched VM